e2e_tests/integration/user.spec.js

```diff
@@ -18,10 +18,10 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-/* global Cypress, cy, test, expect, before */
```

Comment:
so were `test, expect` being accidentally added as they were already in the eslintrc globals?

Reply:
yeah, I'm not sure why we needed to declare those before, probably due to an old ESLint version or something.